distribution/ddf-common/src/main/resources/bin/setenv

```diff
@@ -79,6 +79,9 @@ export DDF_ON_ERROR="bin/ddf_on_error.sh script "
 # Defines the special on-error Java options
 JAVA_ERROR_OPTS="-XX:OnOutOfMemoryError=\${DDF_ON_ERROR}%p -XX:OnError=\${DDF_ON_ERROR}%p"
 
+# Defines java garbage collection logging options
+GC_OPTS="-Xlog:gc=info:file=${DDF_HOME}/data/log/gc.log:time,uptime,level,tags:filecount=5,filesize=50m"
```

Comment:
This is how Solr sets up its GC logging which worked well for us in production.
https://github.com/apache/lucene-solr/blob/3e2fb59272f5b4d8106b3d8edf847f50bacd7a61/solr/bin/solr#L2018

And this is how they configured their log rotation.
https://github.com/apache/lucene-solr/blob/3e2fb59272f5b4d8106b3d8edf847f50bacd7a61/solr/bin/solr#L2031